vcs/displayplot.py

```diff
@@ -108,6 +109,30 @@ def _repr_png_(self):
         f = open(tmp, "rb")
         st = f.read()
         f.close()
+        try:
+            import IPython.display
+            import cdat_notebook
+            if self.g_type == "boxfill":
+                box = vcs.getboxfill(self.g_name)
+                b_dict = vcs.utils.dumpToDict(box)[0]
+                if self._widget is not None:
+                    self._widget.close()
+                self._widget = cdat_notebook.GMWidget(value=b_dict)
```

Comment:
followed the instructions to enable cdat_notebooks-widget extension but the above line still return None for self._widget

Reply:
@aashish24 @downiec @scottwittenburg if you have some time to look into this it would be a good stsarting point for v2.0

Reply:
repo is at: https://github.com/CDAT/cdat-notebook-widgets